.github/workflows/Vulnerability.yml

```diff
@@ -0,0 +1,57 @@
+# name: Vulnerability
```

Comment:
Why the code in this file all commented?

Reply:
There are many vulnerabilities now, maybe we will change the tech stack, 
If we change the tech stack, we can enable it quickly
Just want to save time for the next changes of tech stack

Reply:
Got it!